srcpkgs/apparmor/files/profiles/usr.bin.dhcpcd

```diff
@@ -26,9 +32,9 @@ profile dhcpcd /{usr/,}bin/dhcpcd {
   /proc/sys/net/ipv{4,6}/neigh/*/retrans_time_ms w,
   /proc/sys/net/ipv{4,6}/neigh/*/base_reachable_time_ms w,
 
-  /{var/,}run/dhcpcd{-*,}.pid rwk,
-  /{var/,}run/dhcpcd.sock rw,
-  /{var/,}run/dhcpcd.unpriv.sock rw,
+  /{var/,}run/dhcpcd/pid rwk,
+  /{var/,}run/dhcpcd/sock w,
+  /{var/,}run/dhcpcd/unpriv.sock w,
```

Comment:
Why remove read permissions? Seems like it is harmless to keep them, and if they turn out to be necessary then we do not have to bump the apparmor package again. Also you need to handle interface specific sockets (e.g. `/run/dhcpcd/eth0.sock`). The following should be good:

```
   /{var/,}run/dhcpcd/{,*.}sock rw,
```

Reply:
> Why remove read permissions? Seems like it is harmless to keep them, and if they turn out to be necessary then we do not have to bump the apparmor package again.

It worked fine without so I guessed why not, but I agree, not an issue to keep it in and might spare a revbump.